src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEmbeddedNullableMembersAttributeSymbol.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class SynthesizedEmbeddedNullableMembersAttributeSymbol : SynthesizedEmbeddedAttributeSymbolBase
```

Comment:
>SynthesizedEmbeddedNullableMembersAttributeSymbol [](start = 26, length = 49)

I'll skip reviewing this file in this iteration, since changes are planned